handler/gitlab.go

```diff
@@ -33,9 +33,26 @@ func (g *Gitlab) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
 			return err
 		}
 
-		// Filter events by "MergeRequest" opened
-		if mr.EventType != "merge_request" || mr.ObjectAttributes.Action != "open" {
-			fmt.Fprint(writer, "We just care about new merge_requests")
+		if mr.EventType != "merge_request" {
+			fmt.Fprint(writer, "We just care about merge_request events")
+			return
+		}
+
+		for _, data := range g.provider.GetReactionRules() {
+			if mr.ObjectAttributes.Action == data.Action {
```

Comment:
I rather prefer to continue the loop to allow us to remove an indentation level and have a slightly more readable block.
```
if mr.ObjectAttributes.Action != data.Action {
    continue
}
````